csrankings-w.csv

```diff
@@ -15,6 +15,7 @@ Wadee S. Alhalabi,King Abdulaziz University,http://www.kau.edu.sa/Contacts.aspx?
 Wafaa El-Haweet,King Abdulaziz University,http://www.kau.edu.sa/Content.aspx?Site_ID=61117&lng=EN&cid=60829,NOSCHOLARPAGE
 Wagner Machado Nunan Zola,UFPR,https://www.inf.ufpr.br/wagner,3FAaX00AAAAJ
 Wagner Meira Jr.,UFMG,http://homepages.dcc.ufmg.br/~meira,oad6_oEAAAAJ
+Wai Lam,Chinese University of Hong Kong,https://www1.se.cuhk.edu.hk/~textmine,ewA4NAcAAAAJ

```

Comment:
Please provide evidence that this faculty member can solely advise PhD students in the CS department.